web-console/console-config.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+window.consoleConfig = { /* future configs may go here */ };
```

Comment:
I am imagining that this would be a default and there would be a Druid property that has a JSON object it in that Druid will stringily and prepend with `window.consoleConfig = ` and send instead of this file.

Reply:
I do not see a need to add default configs here as this IMO is just a stepping stone PR towards what I said above. I will add a sweet interface for it though.